assets/stories/skate-components/map/markers/locationDotMarker.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import React from "react"
+import { LocationMarker } from "../../../../src/components/mapMarkers"
+import { inMapDecorator } from "../../../../.storybook/inMapDecorator"
+
+const location: LocationSearchResult = {
```

Comment:
**suggestion:** not sure how much luck you've had working through the CI errors you're seeing, but adding `import { LocationSearchResult } from "../../../../src/models/locationSearchResult"` will help (and then you just have to add location to the args for the two specific stories).

Reply:
I originally had that, but that led to this kind of meaningless input:
![Screenshot 2023-11-14 at 10 49 22 AM](https://github.com/mbta/skate/assets/912020/7dedf13d-4514-4920-8b89-0f0ba54984aa)

I guess it could be helpful for showing usage, but on the storybook page, that input doesn't have any effect and looks kind of ugly.